North-Seattle-College/ad440-winter2022-thursday-repo
# Commits deployStackS3Bucket.py and stackTemplate.json files

**Completes Issue #67** 

### **Description:**
- Modifies automation scripts to create stack and s3bucket, so that they can be run by computer and in github action.
Also, updates the directory name to `DeployCloudformationS3Bucket`.
- The `deployStackS3Bucket.py` script runs using the input parameters provided to be used in naming the Stack and creates S3Bucket. 
- The `stackTemplate.json` file provides parameters used in creation of a new Stack and S3Bucket to Cloudformation.
- Region is default to `us-west-2`.
_______________________________________________________________________________________________________________________________________
### **Prerequisites:**

- Python Boto3 Installed
- AWS Credentials and Config Files 
`/.aws/credentials`
`/.aws/config`

- Install Python Boto3:
`python -m pip install boto3`

- Setup Credentials and Config Files: 
`aws configure`
_______________________________________________________________________________________________________________________________________
### **Testing:**

1. Run the deployStackS3Bucket.py file from command prompt and include initials as a required parameter. 
    Example: `Python deployStackS3Bucket.py mkg`

2. A Confirmation Message Will Appear After New Stack And S3Bucket Are Successfully Deployed to Cloudformation.
3. Verify Your Stack and S3Buckets Are Created in the AWS Management Console Website.
`https://us-west-2.console.aws.amazon.com/console/home`
